client/src/modules/fundingSources/modal/createUpdate.html

```diff
@@ -0,0 +1,47 @@
+<form name="fundingSourceForm" bh-submit="ModalCtrl.submit(fundingSourceForm)" novalidate>
+    <div class="modal-header">
+      <ol class="headercrumb">
+        <li>
+          <span translate>TREE.FUNDING_SOURCES</span>
+
+          <label class="label label-warning text-uppercase" translate>
+            {{ModalCtrl.action}}
+          </label>
+        </li>
+      </ol>
+    </div>
+
+    <div class="modal-body">
+      <div class="form-group clearfix" ng-class="{ 'has-error' : fundingSourceForm.label.$invalid && fundingSourceForm.$submitted }">
+        <label class="control-label" translate>FORM.LABELS.NAME</label>
+        <input type="text" class="form-control col-md-9" name="label" ng-model="ModalCtrl.data.label" required>
+        <div class="help-block" ng-messages="fundingSourceForm.label.$error" ng-show="fundingSourceForm.$submitted">
+          <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+          <p translate> FORM.VALIDATION.FUNDING_SOURCE_NAME_ERROR </p>
+        </div>
+      </div>
+
+      <div class="form-group clearfix" ng-class="{ 'has-error' : fundingSourceForm.code.$invalid && fundingSourceForm.$submitted }">
+        <label class="control-label" translate>FORM.LABELS.CODE</label>
+        <input type="text" class="form-control col-md-9" name="code" ng-model="ModalCtrl.data.code" required>
+        <div class="help-block" ng-messages="fundingSourceForm.code.$error" ng-show="fundingSourceForm.$submitted">
+          <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+          <p translate> FORM.VALIDATION.FUNDING_SOURCE_CODE_ERROR </p>
```

Comment:
![image](https://github.com/user-attachments/assets/9bb343a1-5197-4a28-a2a8-e3ab29d3e1a2)

Both of these are just grey text.  Could you update these to make fields with errors red?  Here is an example from patient group management:

![image](https://github.com/user-attachments/assets/f639ed4d-aeb2-4e51-bdba-8fc247e4e2ec)